addons/mail/static/src/components/message/message_tests.js

```diff
@@ -36,6 +36,7 @@ QUnit.module('message_tests.js', {
                 data: this.data,
             }));
             this.env = env;
+            this.env.session.currencies = {1: {symbol: '$', position: 'before'}};
```

Comment:
Don't define this here, it is either not generic enough, or too generic.

Alternatives are:
- if it's test specific (I think it is): override `env` in `start` of each test that needs it
- if it's global: add in `makeTestEnvironment` in `web`